tests/examples/pyproject_matrix/dependencies.yaml

```diff
@@ -0,0 +1,26 @@
+files:
+  py_build:
+    output: pyproject
+    pyproject_dir: output/actual
+    matrix:
+      cuda: ["11.5", "11.6"]
```

Comment:
When I run `dfg` against this file locally, I get the expected `ValueError: matrix entries are not supported for pyproject.toml files.` error, but I notice that this example isn't under `tests/examples/invalid`.

Is this example supposed to work?

Reply:
Oh, I see now. This example is listed in the `test_error_examples` test case.

These tests might warrant some clean up at some point. But we can save that for another day.

Reply:
Yeah exactly. I added some comments to test_examples.py explaining the difference between invalid and erroneous tests right now, but it could use some more systematic improvement down the line.